drivers/flash/Kconfig

```diff
@@ -162,4 +162,15 @@ config SOC_FLASH_MCUX_DEV_NAME
 	help
 	  Specify the device name for the flash driver.
 
+config DRV_SOC_FLASH_WRITE_BLOCK_SIZE
+	int "write block size of the SoC driver (fixed)"
```

Comment:
int "Minimum write block size supported by the driver"

(not sure what you mean by 'fixed')